casepro/msgs/migrations/0050_auto_20160712_1519.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('msgs', '0049_faq'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
If possible please try to limit migrations within the same PR (e.g. in this case one could amend the original migration). I know this is tricky when code has already been deployed, but where we can do this it helps keep the number of migrations down. Also useful to ensure migrations have meaningful names tho I appreciate sometimes that's not worth it for migrations that do a lot of different things.